pom.xml

```diff
@@ -188,6 +189,7 @@
                 <artifactId>commons-io</artifactId>
                 <version>2.6</version>
             </dependency>
+            <!-- TODO: Robert manages v3.9 but antenna manages 3.8.1 -->
```

Comment:
Do you want to keep these todos? I guess it doesn't hurt to use the older version. I'm just a fan of keeping stuff updated ;)